rebasehelper/specfile.py

```diff
@@ -147,6 +147,16 @@ class SpecContent:
         '%transfiletriggerpostun',
     ]
 
+    # Comments in these sections can only be on a separate line.
+    BLACKLIST_COMMENT_IN: List[str] = [
```

Comment:
I don't like the name. The same goes for `comment_allowed` variable below. I don't think it's exact and the meaning isn't obvious. Let's define the term **inline comment** for a comment that doesn't start at the beginning of a line. Then you can name this attribute e.g. `DISALLOW_INLINE_COMMENTS` and the variable `inline_comment_allowed`.